docs/api/components/schemas.yaml

```diff
@@ -2512,3 +2512,28 @@ CollectionNames:
           - Starting
           - Stopping
           - Resyncing
+All_user_channels_response:
```

Comment:
I would name this `all_user_channels` since otherwise it seems a bit confusing to have two places that refer to the same object.